classify.py

```diff
@@ -313,58 +335,121 @@ def evaluate_samesource(desc, dataset, n_frequent_words, tokens_per_sample, max_
     LOG.info(f'{desc}: number of speakers: {np.unique(y).size}')
     LOG.info(f'{desc}: number of instances: {y.size}')
 
-    X = preprocessor.fit_transform(X)
+    X = preprocessor.fit_transform(X) # shouldn't this take place on the x_train and then on x_test?
 
     lrs = []
+    lrs_voc = []
+    lrs_comb = []
+    lrs_features = []
     y_all = []
-    for X_train, y_train, X_test, y_test in tqdm(get_batch_simple(X, y, repeats, max_n_of_pairs_per_class)):
-        # fit a classifier and calculate LRs
+
+    for X_train, y_train, X_test, y_test, X_voc_train, X_voc_test in tqdm(
```

Comment:
confusing order: train, train, test, test, train, test
I would expect either train, train, train, test, test, test or train, test, train, test, train, test

Reply:
changed to [train sets], [test sets]